sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala

```diff
@@ -24,9 +23,9 @@ import org.json4s.jackson.JsonMethods._
   * @author Vlad Soykrskyy
   */
 class SQLTask(val filepath: String, val taskId: String){
-  private val appConfig = ConfigFactory.parseFile(new File(loadFactoryConfigPath()))
+  logger.info(filepath)
```

Comment:
If you logg information, try to add some context like `logger.info("SQL task for file: " + filepath)`